data/plugins/autocomplete.lua

```diff
@@ -647,12 +647,15 @@ command.add(predicate, {
     reset_suggestions()
   end,
 
-  ["autocomplete:previous"] = function()
-    suggestions_idx = math.max(suggestions_idx - 1, 1)
+  ["autocomplete:previous"] = function()    
+    suggestions_idx = suggestions_idx - 1
+    if suggestions_idx < 1 then
+      suggestions_idx = #suggestions
+    end
```

Comment:
Use the modulo here too.

Reply:
I tried it and it wasn't working😅. So, how?